dev-libs/efl/metadata.xml

```diff
@@ -5,6 +5,14 @@
 	<email>enlightenment@gentoo.org</email>
 	<name>Project Enlightenment</name>
 </maintainer>
+<maintainer type="project">
+	<email>proxy-maint@gentoo.org</email>
+	<name>Proxy Maintainers</name>
+</maintainer>
+<maintainer type="person">
```

Comment:
Reverse the order, proxy-maint always goes last.

Reply:
Fixed